libs/logic-apps-shared/src/designer-client-services/lib/standard/connection.ts

```diff
@@ -410,6 +410,30 @@ export class StandardConnectionService extends BaseConnectionService implements
     });
   }
 
+  private validateLogicAppIdentity(baseUrl: string, identity: Identity | undefined) {
+    if (!isHybridLogicApp(baseUrl) && !isIdentityAssociatedWithLogicApp(identity)) {
```

Comment:
nit let's reverse this so we don't check isHybridLogicApp twice:

```
private validateLogicAppIdentity(baseUrl: string, identity: Identity | undefined) {

    if (isHybridLogicApp(baseUrl) && (!identity?.principalId || !identity?.tenantId)) {
      throw new Error(
        intl.formatMessage({
          defaultMessage: 'App identity is not configured on the logic app environment variables.',
          id: 'WnU9v0',
          description: 'Error message when no app identity is added in environment variables',
        })
      );
  }
  if (!isIdentityAssociatedWithLogicApp(identity)) {
    throw new Error(
      intl.formatMessage({
        defaultMessage: 'A managed identity is not configured on the logic app.',
        id: 'WnU9v0',
        description: 'Error message when no identity is associated',
      })
    );
  }
}
```

Reply:
done